### PR TITLE
Add rvm edt

### PR DESCRIPTION
**Pull request by godiarksel**
Configured the add, remove and edit functions into the application. This allows a user to modify the tasks entered into the list whilst their changes are stored and updated in the localStorage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -43,6 +63,13 @@ const addTasks = (task) => {
...
   completeTask(checkBox);
   addToLocalStorage(task, false, checkBox.length);
+
+  const editEllip = document.querySelectorAll('.fa-ellipsis-vertical');
+  editEllip.forEach((i) => {
+    i.addEventListener('click', () => {
+      editTask(i.previousElementSibling, tasksEl);
+    });
+  });
 };
```
Comment (prolajumokeoni) [inline src/index.js]: - [ ] Kindly implement the function for deleting a task (remove an element from the array)  and the function for editing task descriptions correctly, user  can still edit or see the checkbox icon after delete, this should be cleared after a task is deleted, on the first click the task is not deleted 👍🏿 
-  see the video below  ⬇️ 




https://user-images.githubusercontent.com/69638013/173021768-680f0177-dcfd-4eb5-b8fa-5439089561eb.mp4

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @godiarksel, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -13,41 +13,140 @@ class Task {
...
+  const tasks = document.querySelectorAll('span');
+  for (let i = 0; i < tasks.length; i += 1) {
+    if (tasks[i].classList.contains('completed-tasks')) {
+      localTasks[i].completed = true;
+    } else {
+      localTasks[i].completed = false;
+    }
+  }
+  localStorage.setItem('taskList', JSON.stringify(localTasks));
+};
```
Comment (youmari) [inline src/index.js]: - [ ] When I make a task  done (`completed: true`) and reload the page it becomes again undone `completed: false` it should be saved in the local storage everytime you change the checkbox please try to fix that

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @godiarksel , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Great title and short summary
✔️ Good commit messages
✔️ No linter errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keneogu — APPROVED** [review]
Hi @godiarksel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**